packages/js/components/src/experimental-tree-control/hooks/use-tree.ts

```diff
@@ -10,7 +10,7 @@ import { TreeProps } from '../types';
 export function useTree( {
 	items,
 	level = 1,
-	role = 'tree',
+	role = 'listbox',
```

Comment:
While testing with screen reader, I found that it behaves more intuitively when using 'listbox' role.